### DISCUSSION TITLE
Welcome to Safe-Yelli Discussions!

### DISCUSSION THREAD

**Discussion by rungdung** [rungdung/Safe-Yelli#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HarshaReddyV** [comment]
Hi, great to see a concept I been thinking about in action. 
I would like to see this project expanded to all over India and includes issues such as Corruption, whistle blowing on scam call centres. 
Includes a portal for the police to contact victims to resolve issues while the victim stay anonymous unless choose not to. 

Can you tell me more about the tech used..?  framework, hosting. 

I majorly do development in Django but looking forward to learn and build stuff in JS. TIA

**Reply by rungdung** [reply]
Hello Harsha,

Glad to have you here. You can absolutely clone the repo for other use cases.

The site is built using R, layouted/templated using the Blogdown package which uses Hugo. The form is a Netlify Form which links to Google Sheets. Periodically the sheet is pulled into the repo (automatically). The site is however currently manually built in Blogdown and pushed due to an unfixed Bug in the Github Actions workflow.

The stack has obviously grown old and is getting complicated to maintain. We are considering rewriting in Svelte. We welcome any PRs, if you want to collaborate on another aspects do drop an email at safeyelli@gmail.com
